driver-core/pom.xml

```diff
@@ -334,6 +334,13 @@
               <classpathDependencyExcludes>
                 <classpathDependencyExcludes>io.netty:netty-transport-native-epoll</classpathDependencyExcludes>
               </classpathDependencyExcludes>
+              <properties>
```

Comment:
I followed the pattern in driver-extras and I also did some refactoring to declare all the profiles in the parent pom and have the child profiles extend from them.  It seems to work very well and removes a lot of duplication in our poms, but need to give CI a spin to see if it behaves as we'd like.